### ISSUE TITLE
make UBI income tax free and cap taxes to 50%

### ISSUE THREAD

**Issue by DanielKispert** [D3nnis3n/White-Tiger-Feedback#35]
right now UBI income can be taxed by the federal government, leading to a money stream from the federal bank to the federal government while the citizen does not interact with that money. The federal government should tax things the people do (e.g. sell things, or earn money through interacting with each other) and not pull straight from the UBI and skipping the citizens. 

Therefore I propose to make the UBI income exempt from all taxes (if the fed bank wants to send more money to the federal government they should increase the seigniorage). This can be achieved by e.g. adjusting a wealth tax so it does not tax below the amount of UBI a citizen would receive (easy because this is always the same, hourly UBI * hours of server uptime) and adjusting income taxes so UBI income (payments from fed bank) do not count into it

Secondly, we should mention somewhere in the constitution that taxes should be no higher than 50% of the thing that is being taxed, based on a real example I think? (unsure how to handle a wealth tax in this case)

**Comment by Tralolol** [comment]
hear ye hear ye!

**Comment by Bazookar** [comment]
This is a great idea, except it's not feasible with the current dynamic of euros. The taxes are only as legitimate as the euros themselves, which lacks value. Players don't care about how many others log onto the server, logging on doesn't generate tradable value in eco, therefore it feels like printing money. 

The government needs to shoulder near the entire amount of circulation through university by day 14 (for reference, roughly 1.6m required for Uni, 2.2m euros in circulation by around day 14). 

To cap the amount of UBI taxable would lead to a donation-centric economy system because it will be near impossible to generate enough revenue. We have seen this in WT 24, where one sales tax cannot support two times the euros in circulation. WT24 relied on players donating (buying lasers) to finance the end game goal. Note that WT23 the uni bills were 'free' for the government. 

Increasing UBI could solve the problem - because there will be more euros to tax. But it may cause inflation because the money is directly placed into players' hands - making players, especially new players who don't subscribe to a general pricing system in the past, feel richer. An example of this is new players purchasing the 200-500 euro millionaire pizza with their UBI.

With UBI instead of the university method, the government lacks influence on pricing. If it is a university system, the university can direct the pricing of goods in hope of keeping inflation in check because the university is the *only* way that players can obtain currency. i.e. if a player sells 5 ores to university for 0.1 euros, players would not likely spend that 0.1 euro to buy only 1 ores; therefore, ores would likely be 0.02 euros. 

With the UBI system, the university follows the market price and faces pressure from players to increase their price. Thus, increasing UBI and circulation may help temporarily until the market adjusts for inflation. More euros in circulation - more euros to tax. But price will increase, and the more euros taxed would have similar value as before.

**Comment by Bazookar** [comment]
Player activities typicall are:
1. gather
2. trade
3. build
4. craft
5. Talk
6. Run around

Tax based on activity would tax one of the 6 above. Not sure how to tax running around. Labour/crafting tax currently isn't possible (or is it with new thomasfn mod?). Gathering (mining/logging etc) tax is one of the last things players want to tax because those are the driving force of server progression. 

You are now left with sales tax + some forced sale mechanism (income VAT tax or whatever) to keep players' cash assets low. Whether this alone supports uni + end game goals in euros is difficult to evaluate because every successfully funded WT consists of a solid sum of resources donated with a variety of taxes. But what we do know and have seen is that a single sales tax probably does not work.

**Comment by DanielKispert** [comment]
![grafik](https://user-images.githubusercontent.com/34270661/139944591-726fe453-c054-40a6-ae17-f995dd605b74.png)

**Comment by D3nnis3n** [comment]
@DanielKispert You didn't even answer Bazookar, before I approve this that would be the very least that could be expected, no?

**Comment by DanielKispert** [comment]
true, trying to answer all points here with my opinion:

1. don't know what the money faucet has to do with a "dynamic" or why it becomes not feasible. What is value even and why are Euros lacking them. An euro is 1/Xth of an hour of a player existing on the server. "it feels like printing money" -> yes, Euros are entering the circulation through UBI

2. yes the university costs a lot of money. People can choose to donate there too, and the government has to find a good tax plan to do all the rest. It certainly is in the interest of the players to advance research. If they don't because "someone else will be dumb enough to sell instead" then maybe everyone as a community failed, but uni problems also isn't the point of UBI

3. the government also using donations to fund is intended, we know that it's super hard to get enough income otherwise

4. increasing UBI is only betting on stupid players still setting the same prices without realizing that their money is worth less now. That's the same as that constant wealth tax that is just mimicking inflation into the governments pockets.

5. again university problem, not UBI problem

6. another university problem, they are supposed to buy below intrinsic value for that reason. If someone doesn't sell at the uni because they get 5€ more somewhere else but in return their research is screwed, that's a behavior problem, not a balance problem. 

2nd message:
labor tax is possible but gets you bonked. Also another government funding complaint

=> in total, use proper methods of taxation or rely more on donations. The government is supposed to tax things that the citizens notice. If the result of a tax is "oh I just get less UBI because some is always drained to the government" then that is cheap. If we wanted that, we would let the main government mint euros and be done with it. 
If this change results in higher taxes, or a property tax, or an existence tax or whatever, that is all fine (for this purpose, not necessarily for the purpose of the server rules). A daily property tax or daily sewage fee or whatever can still take away all of the players money, including his UBI earnings, since then the citizen actually gets "hurt" by it. That is the main point of this change for me, the government needs to tax the citizens that play in ways that impact them, and not try to always find ways around it (e.g. tax abandoneds, print money, continuously drain UBI) 

The government doesn't provide any value or progress on its own, it can only organize things. Only players can actually play and progress. Some of that value and progress (€€€) has to flow from players to the government (taxes) so they can promise it back to them in exchange for goods or services so common goals can be reached (e.g. university). If players don't see the necessity in that, the server fails and they hopefully learn a bit.

**Comment by Bazookar** [comment]
Since I've said that this is a fine adjustment, I'll respond to only one point for now. My point is not that this can't work, but rather, I find it to be a backwards solution. I specifically respond to “Leading to money stream from federal bank to the treasury”. The statement speaks of a flaw of the UBI, and not the tax. Hence I say it is a backward solution. The issue is that UBI coming from fed bank.

*main point 1* If designer want UBI to be a real benefit, it has to come from taxes. Short of that, it's, again, just an automated version of players printing money. 

Responses to point 4
--------- about people being stupid
I'll specific respond to point 4 where players are called stupid. First, that's not the case. Even if it is, I would be respectful to others. I'd feel quite embarrassed if at some point those allegations become wrong, and now I am the stupid one

It is well known in economics that there is a lag from money to price. That means when the amount of money grows; it takes time for the economy to adjust. 

https://files.stlouisfed.org/files/htdocs/publications/review/80/10/Lag_Oct1980.pdf Disclaimer: publisher not checked, peer review not checked. But it is a paper stating word for word on this one phenomenon. 

If players did not adjusting their price to money immediately, it is lag phenomenon in economics. 

-------- about euro circulation 
Perhaps I'm not clear enough in stating that price will inflate absent your suggestion of not making UBI taxable (your suggestion), but now you have it. When saying increasing UBI could solve the problem, that solution is not a “simple turn up the tap”.

*main point 2*  Think of it this way, the total functional currency is the amount outside of hermits/inactives’ hands that are being used. 

When the players have this concept, then, they will understand that the players who took chunks of UBI and become inactive but isn't yet abandoned, cause the statistic “euros in circulation” to increase, but those euros are not actually available (because now no one can touch those fEuros). Your suggestion has theeffect of reducing euros in circulation.

It's in that context that I suggested turning up UBI. Absent your suggestion being implemented I would recommend keeping it the same or even reduce it.
------------ about fake tax inflation 
There is to some degree an inflation for any tax. But these aren’t fake, and it’s not the same as inflation caused by money.

*main point 3* The reason why it’s not inflation and does not cause significant inflation is: players are given a method where they can reduce the tax burden. 

I describe 3 taxes in the past 3 cycles:
Tax1: 24 hour track custom stat wealth, if it is higher than last tick, tax a percentage
Tax2: give rebate when buying from store, tax when earning
Tax3: hourly tax on wealth for a progressive percentage

All three taxes have one common theme: they can be reduced to some extent, with tax1 and tax3 being potentially (though difficult) fully nullified.

Tax1 and Tax3 do not cause inflation because if a player increases their price because of the tax, some of their competitors will decide not to increase their price but instead spend more euros. 

Tax2, as I have previously illustrated in WT chat, has an effect of raising sales tax by a small amount. When players understand this concept, it would increase their price by the small amount to “transfer tax to the consumer”

Theoretically, any tax can be transferred to consumer by charging a larger margin. This transfer generally does not happen as much to tax1 and tax3 because there is a way to reduce it. And competitors will do it. 

Or in the cooks silent agreement case, we all decided to not spend and keep large margins (see cooks cartel comment in feedback thread, and see stubborn #1 taxed stat). Cooks have nothing to spend on after stoves anyways. 

Disclaimer: there were price adjustments for food stuff, just not much. My personal reasoning is that advanced food will have no market later, so margin has to be big. I think my guess was correct.
—————
TL;DR: read 3 main points

**Comment by DanielKispert** [comment]
closing since it was done
